tokenmagic/module/settings.js

```diff
@@ -624,10 +620,10 @@ Hooks.once("init", () => {
       }
     }
 
-    if ( wmtRefreshTemplate ) {
-      if ( wmtRefreshTemplateType && wmtRefreshTemplateType !== "OVERRIDE" ) {
+    if (wmtRefreshTemplate) {
+      if (wmtRefreshTemplateType && wmtRefreshTemplateType !== "OVERRIDE") {
         const cmtRefreshTemplate = MeasuredTemplate.prototype._refreshTemplate;
-        MeasuredTemplate.prototype._refreshTemplate = function() {
+        MeasuredTemplate.prototype._refreshTemplate = function () {
           return wmtRefreshTemplate.call(this, cmtRefreshTemplate.bind(this), ...arguments);
         };
       }
```

Comment:
The PR is difficult to read because you are using a different code style. Could you fix that please?

Reply:
![image](https://github.com/Feu-Secret/Tokenmagic/assets/52590783/6626f598-5729-42f2-bdf3-55a34b3b36f1)